src/Symfony/Component/Security/Tests/Http/Session/SessionAuthenticationStrategyTest.php

```diff
@@ -39,6 +39,10 @@ public function testUnsupportedStrategy()
 
     public function testSessionIsMigrated()
     {
+        if (PHP_VERSION_ID >= 50400 && PHP_VERSION_ID < 50411) {
+            $this->markTestSkipped('We must not destroy the old session on php 5.4.0 - 5.4.10.');
```

Comment:
Well, but we _can_ destroy the session. We would just run into trouble afterwards. :wink: Okay, I will change the message.

Yes, I can merge those tests again. I made two tests for the following reasons:
* I'd like to keep logic (`if`/`else`) out of tests.
* They are testing different behavior.
* The second test can simply be dropped as soon as php 5.4 is not supported anymore.